.travis.yml

```diff
@@ -14,4 +14,8 @@ matrix:
 script:
   - cargo test --all
   - cargo clean
-  - ./publish-wasm.sh
+  - ./init.sh
+  - ./build.sh
+  - if [ "$TRAVIS_PULL_REQUEST" != "true" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
```

Comment:
Is there any way to test this condition without merging the PR?

Reply:
Locally set GH_TOKEN and then manually run the publish script just to make sure it goes through. (You may not have access to the repo, will add you there). At least it will make sure the publish part works (The above condition itself should)